lib/parseOptions.js

```diff
@@ -16,6 +16,7 @@ const SUPPORTED_ENTITY_TYPES = [
 ]
 
 export default function parseOptions (params) {
+  // TODO: Do we need defaults here for timeout and retryLimit?
```

Comment:
I think no. We should let contentful-management sdk provide its own defaults, otherwise we'll always override them with whatever defaults we set here, which could be confusing in the future